### DISCUSSION TITLE
GCM using too much CPU

### DISCUSSION THREAD

**Discussion by surya-prakash-susarla** [git-ecosystem/git-credential-manager#999] [Q&A] [answered]
GCM has been causing CPU usage spikes and is putting high pressure consistently to the point where it is heating up my machine.

Here is a screenshot of the usage:
<img width="913" alt="image" src="https://user-images.githubusercontent.com/26598313/208340098-925be217-aed9-4370-a400-6f814368f452.png">

Is anybody else also seeing this?

PS: My machine is a MBP M2 and this has only started happening recently.

**Comment by ldennington** [comment, marked as answer]
Hi @Surya-06 - apologies for the pain here. Note that we're aware of this and are tracking with this issue:

https://github.com/GitCredentialManager/git-credential-manager/issues/939

**Reply by surya-prakash-susarla** [reply]
Thanks for providing a link to the issue @ldennington ! It has somehow not happened in the past couple of days interestingly and I do not remember doing an update. Good luck anyway.
